documentation/functions/domain/IGNORE.md

```diff
@@ -3,6 +3,102 @@ name: IGNORE
 ts_ignore: true
```

Comment:
Since this function is no longer deprecated, it would be nice to have it in the types again.

Reply:
Done!